File: PHP/Disk.php
<?php

class Disk{
    private $tipe;
    private $size;
    private $price;
    
   

    public function __construct($tipe = 0, $size = 0, $price = 0) {
        $this->tipe = $tipe;
        $this->size = $size;
        $this->price = $price;

    }

    public function setDiskTipe($tipe) { //
        $this->tipe = $tipe;
    }

    public function getDiskTipe() { 
        return $this->tipe;
    }

    public function setDiskSize($size) { //
        $this->size = $size;
    }

    public function getDiskSize() { 
        return $this->size;
    }


    public function setDiskprice($price) { //
        $this->price = $price;
    }

    public function getDiskPrice(){ 
        return $this->price;

    } 

}
?>

File: c++/Disk.cpp
#include<iostream>
#include<string>

using namespace std;

class Disk{


    private :
    string tipe;
    int Storage;
    int Disk_price;


    public :
    Disk(){

    }

    Disk(string jenis, int capacity, int price){
        tipe = jenis;
        Storage = capacity;
        Disk_price = price;
    }

    void setCapacity(int capacity){
        Storage = capacity;
    }

    void setPrice(int price){
        Disk_price = price;
    }

    void setDisk(string jenis){
        tipe = jenis;
    }

    int getDiskCapacity(){
        return Storage;
    }

    int getDiskPrice(){
        return Disk_price;
    }

    string getDisk(){
        return tipe;
    }

    

    ~Disk(){

    }
};
